### DISCUSSION TITLE
Page Previous Button on M5stickV

### DISCUSSION THREAD

**Discussion by odudex** [selfcustody/krux#131] [Ideas]
It is possible([video](https://photos.app.goo.gl/ui5iH4VH4RpgaBdW9)) with some "dirty" code use power button as a previous button.
It's just polling power button state from PMU using I2C.
It loses the "reset" functionality but still shuts down if pressed for ~3 seconds
Any possible problems or cons?

**Comment by ghost** [comment]
Clever! That's a great idea and will make the stick much easier to use, especially for mnemonic input. I can't think of any real downsides. We have a watchdog timer now that will reset the device if Krux hangs or becomes unresponsive, and if you can still shutdown the device by holding it down for a few seconds I don't think anything will really be lost by doing this.

Sounds good to me! Can you make that a separate PR?

**Reply by odudex** [reply]
Sure, will make it in another PR.
It is also necessary to add some lines to `pmu.py`. So I must do a PR in MaixPy too.
I was thinking about update fonts.c for Amigo and Bit too, as previously I only used main ascii characters. I noticed if I increase to 256 it will cover all latin/saxon languages. Do you agree?
